contracts/protocol/facets/ProtocolInitializationFacet.sol

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.9;
+
+import "../../domain/BosonConstants.sol";
+import { IBosonProtocolInitializationHandler } from "../../interfaces/handlers/IBosonProtocolInitializationHandler.sol";
+import { ProtocolLib } from "../libs/ProtocolLib.sol";
+import { ProtocolBase } from "../bases/ProtocolBase.sol";
+import { DiamondLib } from "../../diamond/DiamondLib.sol";
+
+/**
+ * @title IBosonProtocolInitializationHandler
+ *
+ * @notice Handle initializion of new versions after 2.1.0.
+ *
+ */
+contract ProtocolInitializationFacet is IBosonProtocolInitializationHandler, ProtocolBase {
+    /**
+     * @notice Modifier to protect initializer function from being invoked twice for a given version.
+     */
+    modifier onlyUnInitializedVersion(bytes32 _version) {
```

Comment:
as above, i would rename to `onlyUninitializedVersion`

Reply:
done